libse/SubtitleFormats/Pac.cs

```diff
@@ -1546,18 +1547,7 @@ private static byte[] GetW16Bytes(string text, byte alignment, int encoding)
             return result.ToArray();
         }
 
-        private static bool OnlyAnsi(string line)
-        {
-            string latin = Utilities.AllLettersAndNumbers + " .!?/%:;=()#$'&\"";

```

Comment:
There are some character that aren't in **ANSI** range that are present in `Utilities.AllLettersAndNumbers`